### ISSUE TITLE
why the paper is so similar to our project?

### ISSUE THREAD

**Issue by ShineChen1024** [ShineChen1024/oms-Diffusion#33]
https://raywang335.github.io/stablegarment.github.io/

**Comment by ShineChen1024** [comment]
@raywang335

**Comment by xdfeng07** [comment]
indeed, quite similar. especially the end part where they show the improvement of adding densepose controlnet as extra guidance. As far as I know there is a work about controlnet with densepose, i think you could also achieve similar improvements by using this controlnet together with your model.

**Comment by sunnytomy** [comment]
I see what you mean, you may want to raise your concern to the authors.
